core/tests/fixtures/ipfs_folder/hello.txt

```diff
@@ -0,0 +1 @@
+hello world
```

Comment:
Is `hello.txt` being used?

Reply:
I left it just so the fold is not empty, can also be used for other tests but I'm not using it otherwise we could get false results on the tests from previous runs.

Reply:
Makes sense.